stratum/hal/lib/barefoot/barefoot.bzl

```diff
@@ -67,8 +67,20 @@ pkg_tar(
   name = "bf_shareable_files",
   srcs = glob(["barefoot-bin/share/microp_fw/**"]) +
          glob(["barefoot-bin/share/bfsys/**"]) +
-         glob(["barefoot-bin/share/tofino_sds_fw/**"]),
+         glob(["barefoot-bin/share/tofino_sds_fw/credo/**"]) +
```

Comment:
`glob` has an `exclude` [param](https://docs.bazel.build/versions/master/be/functions.html#glob). Could simplify this rule.